kodilla-exception/src/main/java/com/kodilla/exception/test/FlightFinder.java

```diff
@@ -0,0 +1,13 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+
+public class FlightFinder {
+    public void findFlight(Flight flight) throws RouteNotFoundException {
+        HashMap<Flight, Boolean> flights = new HashMap<>();
```

Comment:
Blad jest tutaj. Tworzysz `HashMap<Flight, Boolean> flight`. Nastepnie odwolujesz sie do niej, a jest pusta. 
Ta HashMap'a to inna hashmapa niz tworzona w main w FlightFinderRunner

Reply:
Rozwiazaniem bedzie np. przekazywac w konstruktorze flights w momencie tworzenia FlightFinder
```
    private HashMap<Flight, Boolean> flights;

    public FlightFinder(HashMap<Flight, Boolean> flights) {
        this.flights = flights;
    }

    public void findFlight(Flight flight) throws RouteNotFoundException {
        if (flights.get(flight) != null) {
            System.out.println("Route: " + flight + "is available.");
        }
        throw new RouteNotFoundException("Route not found");
    }

Reply:
+ w main musisz zbudowana hashmape przekazac do FlightFinder